IndexHelper.js

```diff
@@ -69,7 +69,11 @@ export class IndexHelper {
     const blindOps = [];
     for(const [key, unique] of indexes.entries()) {
       const value = content[key];
```

Comment:
I think this would be more clear if `value` was not used in every possible way.  Maybe this could be `contentValue`?

Reply:
Then I wouldn't be able to pass it in as `value` below and I'd be sad.

Reply:
I could normalize to an array instead... but seems like overkill.